control/sisotool.py

```diff
@@ -0,0 +1,144 @@
+__all__ = ['sisotool']
+
+from .freqplot import bode_plot
+from .timeresp import step_response
+from .lti import issiso
+import matplotlib
+import matplotlib.pyplot as plt
+import warnings
+
+def sisotool(sys, kvect = None, xlim_rlocus = None, ylim_rlocus = None, plotstr_rlocus = 'b' if int(matplotlib.__version__[0]) == 1 else 'C0',rlocus_grid = False, omega = None, dB = None, Hz = None, deg = None, omega_limits = None, omega_num = None,margins_bode = True, tvect=None):
+
+    """Sisotool
```

Comment:
Documentation style is a bit nonstandard.  I think the Sisotool on its own line can be removed.  This will look odd on `readthedocs` since the function name will already be listed.